jstests/core/explain_verbosity_unknown.js

```diff
@@ -0,0 +1,22 @@
+//Test whether unknown commands are accepted in explain function
+
+(function() {
+	
+	"use strict";
+	
+	var unknownCollection = db["jstestDB"];
+
+	//insure the collection has an item
+	unknownCollection.insert({a:1});
+
+	//check with finding item that exist
```

Comment:
How about `// Test that the explain command rejects unknown options when the collection exists.`?